--- bootstrap.orig	Tue Nov 23 10:19:34 1999
+++ bootstrap	Tue Nov 23 10:18:45 1999
@@ -8,7 +8,7 @@
 
 echo    Building the runtime-library in ./rss
 cd rss
-gcc -c -O -DHAVE_GLOB *.c
+gcc $CFLAGS -c -DHAVE_GLOB *.c
 ar rsv libicrss.a *.o
 rm *.o
 cd ..
@@ -18,23 +18,23 @@
 
 echo Creating icmake
 cd make
-gcc -O -DHAVE_GLOB -o ../bin/icmake *.c ../rss/libicrss.a
+gcc $CFLAGS -DHAVE_GLOB -o ../bin/icmake *.c ../rss/libicrss.a
 
 echo Creating icm-pp
 cd ../pp
-gcc -O -DHAVE_GLOB -o ../bin/icm-pp *.c ../rss/libicrss.a
+gcc $CFLAGS -DHAVE_GLOB -o ../bin/icm-pp *.c ../rss/libicrss.a
 
 echo Creating icm-comp
 cd ../comp
-gcc -O -DHAVE_GLOB -o ../bin/icm-comp *.c ../rss/libicrss.a
+gcc $CFLAGS -DHAVE_GLOB -o ../bin/icm-comp *.c ../rss/libicrss.a
 
 echo Creating icm-exec
 cd ../exec
-gcc -O -DHAVE_GLOB -o ../bin/icm-exec *.c ../rss/libicrss.a
+gcc $CFLAGS -DHAVE_GLOB -o ../bin/icm-exec *.c ../rss/libicrss.a
 cd ../un
 
 echo Creating icmun
-gcc -O -DHAVE_GLOB -o ../bin/icmun *.c ../rss/libicrss.a
+gcc $CFLAGS -DHAVE_GLOB -o ../bin/icmun *.c ../rss/libicrss.a
 cd ..  
 
 echo Stripping the binaries
